internal/cli/atlas/serverless/list.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serverless
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var listTemplate = `ID	NAME	MDB VER	STATE{{range .Results}}
```

Comment:
[Question for my own understanding] Should there be space in `MDB VER`? I see Result have 4 fields but header have 5 (ID, NAME, MDB, VER, STATE). I was thinking if it should be `MDB_VER` instead of `MDB VER`?

Reply:
this was based on https://github.com/mongodb/mongocli/blob/master/internal/cli/atlas/clusters/list.go#L40

Reply:
ah okay, maybe @andreaangiolillo could answer this question.

Reply:
I am not sure why it seems that there is a space between `VER` and `STATE` but in reality, there is a tab

Reply:
Thank you!